autoload/neomake/makers/ft/go.vim

```diff
@@ -47,3 +48,17 @@ function! neomake#makers#ft#go#govet()
             \ '%-G%.%#'
         \ }
 endfunction
+
+function! neomake#makers#ft#go#gometalinter()
+    " Only run a subset of gometalinter for speed, users can override with:
+    " let g:let g:neomake_go_gometalinter_args = ['--disable-all', ....]
+    "
+    " All linters are only warnings, the go compiler will report errors
+    return {
+        \ 'args': ['--disable-all', '--enable=gosimple', '--enable=staticcheck'],
+        \ 'append_file': 0,
+        \ 'cwd': '%:h',
+        \ 'mapexpr': 'neomake_bufdir . "/" . v:val',
+        \ 'errorformat': '%W%f:%l:%c:%m',
```

Comment:
Thanks @novas0x2a, how exactly is this an improvement? (Serious question, isn't meant to sound sarcastic).